package.json

```diff
@@ -230,7 +235,31 @@
                 "category": "Azure Container Apps"
             }
         ],
+        "submenus": [
+            {
+                "id": "containerApps.submenus.workspaceActions",
+                "label": "Azure Container Apps",
+                "icon": "resources/azure-containerapps.svg"
```

Comment:
You need to define the icon paths like this: https://github.com/microsoft/vscode-azurefunctions/pull/3784/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R352-R355

Or else, the icon won't show up in light theme for whatever reason.